osu.Game/Screens/Menu/Intro.cs

```diff
@@ -72,6 +74,8 @@ private void load(AudioManager audio, OsuConfigManager config, BeatmapManager be
 
             welcome = audio.Samples.Get(@"welcome");
             seeya = audio.Samples.Get(@"seeya");
+
+            audio.AddAdjustment(AdjustableProperty.Volume, exitingVolumeFade);
```

Comment:
Can this be added in `OnResuming` instead?  I don't see any particular need for it to exist throughout the lifetime of the game.

Reply:
Resolved, I took it from inactive volume fading implementation, but i realized the usage here is different